layouts/partner/aws.html

```diff
@@ -8,165 +8,80 @@
 
     <section id="overview" class="my-16">
         <div class="flex flex-wrap container mx-auto">
-            <div class="w-full order-last lg:mt-0 lg:order-first lg:w-1/2 px-5">
+            <div class="w-full order-first lg:mt-0 lg:order-first lg:w-1/2 px-5">
                 <h2>Universal Infrastructure as Code for AWS</h2>
```

Comment:
Can you move this into the markdown. Right now the Hero title is there, but this subheading isn't there, so its a bit confusing.